frontend/hub/namespaces/HubNamespaces.tsx

```diff
@@ -43,21 +44,42 @@ export function Namespaces() {
 }
 
 export function AllNamespaces() {
-  return <CommonNamespaces url={hubAPI`/_ui/v1/namespaces/`} />;
+  return <CommonNamespaces url={pulpAPI`/pulp_ansible/namespaces/`} queryParams={undefined} />;
 }
 
 export function MyNamespaces() {
-  return <CommonNamespaces url={hubAPI`/_ui/v1/my-namespaces/`} />;
+  return (
+    <CommonNamespaces
+      url={pulpAPI`/pulp_ansible/namespaces/`}
+      queryParams={{ my_permissions: 'ansible.change_ansiblenamespace' }}
+    />
+  );
 }
 
-export function CommonNamespaces({ url }: { url: string }) {
+export function CommonNamespaces({
+  url,
+  queryParams,
+}: {
+  url: string;
+  queryParams:
+    | {
```

Comment:
```suggestion
    {
```

Reply:
The linter wanted that pipe for some reason. ??

Reply:
hm, interesting, do you have the message?
But maybe it's doing the same thing, I know typescript can do `foo: A | B`  for either-or types, but I've never seen a `foo: | A | B`.

Reply:
Running eslint didn't seem to help much ... 
```
/Users/semighdo/red-hat/ansible-ui/frontend/hub/namespaces/HubNamespaces.tsx
  65:5  error  Insert `|`  prettier/prettier
```

Reply:
Hm, ok, let's believe it, I withdraw my comment :)